### PR TITLE
Feature/804integrate event framework

### PR DESCRIPTION
**Pull request by ndonnan**
Updated to latest version of event sdk and added dependency on the central services logger

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,33 +34,35 @@
...
     "catbox": "10.0.6",
     "catbox-memory": "4.0.1",
     "glob": "7.1.4",
     "immutable": "3.8.2",
     "lodash": "4.17.15",
-    "mustache": "3.0.3",
+    "mustache": "3.1.0",
     "raw-body": "2.4.1",
     "uuid4": "1.1.4",
     "winston": "3.2.1"
```
Comment (rmothilal) [inline package.json]: Please remove Winston dependency

File: package.json [hunk 2]
```diff
@@ -34,33 +34,35 @@
...
   },
   "devDependencies": {
-    "@hapi/hapi": "18.3.2",
+    "@hapi/hapi": "18.4.0",
     "@hapi/joi": "15.1.1",
-    "chance": "1.0.18",
+    "@modusbox/mojaloop-sdk-standard-components": "0.0.39",
```
Comment (mdebarros) [inline package.json]: Any reason we need this?

Comment (ndonnan) [reply package.json]: This dependency has been removed

File: package.json (hunk 1 above)
Comment (ndonnan) [reply package.json]: Removed dependency

**Pull request merged**